aws/app/lambda/reliability/lib/dataLayer.js

```diff
@@ -99,6 +99,9 @@ async function saveToVault(
   const formSubmission =
     typeof formResponse === "string" ? formResponse : JSON.stringify(formResponse);
 
+  const submissionDate = new Date(Number(createdAt));
+  const name = `${('0' + submissionDate.getDate()).slice(-2)}-${('0' + (submissionDate.getMonth() + 1)).slice(-2)}-${submissionID.substring(0, 4)}`;
```

Comment:
Nicely done, I had to wrap my head around this one for a bit as to why we were adding ` + 1` to the `getMonth()` fn.  After looking at the fn definition I'm less confused.  For everyone else `getMonth` returns a number in the range `0-11` instead of the actual numerical representation of the month `1-12`.

Reply:
Oh yes! I guess I could have added a comment to explain why 😕